src/dguweb/web/static/elm/SearchBox.elm

```diff
@@ -182,24 +182,17 @@ viewForm =
                 )
             )
     in
-        Html.form
-            [ action "/search"
-            , method "GET"
-            ]
-            [ Html.fieldset []
-                [ Html.input
-                    [ id "q"
-                    , name "q"
-                    , type' "text"
-                    , placeholder "Search for data"
-                    , class "form-control search"
-                    , autocomplete False
-                    , onInput Lookup
-                    , onWithOptions "keydown" keydownOptions keydownDecoder
-                    ]
-                    []
-                ]
+        Html.input
```

Comment:
There's a way. I'll do it separately https://github.com/datagovuk/dgu2/issues/78